pkg/options/options.go

```diff
@@ -62,6 +63,7 @@ func NewOptions() *Options {
 		Resources:            ResourceSet{},
 		MetricAllowlist:      MetricSet{},
 		MetricDenylist:       MetricSet{},
+		OptInMetrics:         MetricSet{},
```

Comment:
rename to
```suggestion
		MetricOptInList:         MetricSet{},
```
for consistency

Reply:
Renamed it 👍🏼